### PR TITLE
Bug/215

### LINKED ISSUE #215: Posts H2 and other hero styling
Posts hero for author styling and h2 are incorrect on mobile:
The_Essential_Cigar_Advisor_Guide_to_CLE_Cigars_and_The_Essential_Cigar_Advisor_Guide_to_CLE_Cigars

### PR DESCRIPTION
**Pull request by kailasnadh790**
Fix #215 

Test URLs:
- Before: https://main--famous-smoke-cigaradvisor--hlxsites.hlx.page/my-weekend-cigar-review-red-meat-lovers-filet-mignon
- After: https://bug-215--famous-smoke-cigaradvisor--hlxsites.hlx.page/cigaradvisor/category/cigar-reviews/my-weekend-cigar-review-red-meat-lovers-filet-mignon

### CODE REVIEW

File: cigaradvisor/blocks/articleheader/articleheader.css [hunk 1]
```diff
@@ -90,3 +80,33 @@
...
+    .articleheader.block .article-info .article-author {
+        flex-direction: row;
+    }
+
+    .articleheader.block .article-info .article-author>a::after {
+        content: "";
+        display: inline-block;
+        width: 1px;
+        height: 12px;
+        background-color: #3c3a3b;
```
Comment (Kronnox) [inline cigaradvisor/blocks/articleheader/articleheader.css:103]:
```suggestion
        background-color: var(--dark-grey);
```

File: cigaradvisor/blocks/articleheader/articleheader.css [hunk 2]
```diff
@@ -90,3 +80,33 @@
     max-width: 1080px;
 }
 
+.articleheader.block .article-info .article-author > *{
```
Comment (Kronnox) [inline cigaradvisor/blocks/articleheader/articleheader.css:83]: nit:
```suggestion
.articleheader.block .article-info .article-author > * {
```

File: cigaradvisor/blocks/articleheader/articleheader.css [hunk 3]
```diff
@@ -90,3 +80,33 @@
...
+    font-weight: var(--font-weight-semibold);
+    font-size: var(--body-font-size-xs);
+    margin: 5px 0;
+}
+
+@media screen and (min-width: 600px) {
+    .articleheader.block .article-info .article-author > *{
```
Comment (Kronnox) [inline cigaradvisor/blocks/articleheader/articleheader.css:90]: nit:
```suggestion
    .articleheader.block .article-info .article-author > * {
```

File: cigaradvisor/blocks/articleheader/articleheader.css [hunk 4]
```diff
@@ -90,3 +80,33 @@
...
+        margin: 0;
+    }
+
+    .articleheader.block .article-info .article-author {
+        flex-direction: row;
+    }
+
+    .articleheader.block .article-info .article-author>a::after {
```
Comment (Kronnox) [inline cigaradvisor/blocks/articleheader/articleheader.css:98]: nit:
```suggestion
    .articleheader.block .article-info .article-author > a::after {
```

**Review by bstopp — APPROVED** [review]

**Pull request merged**